socrates/test/notifications/notifications_test.js

```diff
@@ -209,25 +209,31 @@ describe('Notifications', () => {
     });
 
     it('waitinglist - creates a meaningful text and subject', () => {
-      notifications.changedWaitinglist(hans, {desiredRooms: [roomOptions.informationFor('junior', 'waitinglist')]});
+      const details = Object.assign({desiredRooms: [roomOptions.waitinglistInformationFor('junior'),
+        roomOptions.waitinglistInformationFor('bed_in_double')]}, roomOptions.nightsUntilFor(3));
+      notifications.changedWaitinglist(hans, details);
       expect(transport.sendMail.calledTwice).to.be(true);
       const options = transport.sendMail.firstCall.args[0];
       expect(options.bcc).to.eql(hansmail);
       expect(options.subject).to.equal('SoCraTes Waitinglist Change of Room Option');
-      expect(options.html).to.contain('junior room (exclusively)');
-      expect(options.html).not.to.contain('nights until');
-      expect(options.html).to.contain('Once we have drawn the lottery and you get a spot, we will book you a room');
+      expect(options.html).to.contain('bed in a double room');
+      expect(options.html).to.contain('<b>3</b>');
+      expect(options.html).to.contain('sunday morning');
+      expect(options.html).to.contain('We will inform you as soon as you got a spot through the lottery or from the waitinglist (in case you registered after the lottery deadline).');
       expect(options.from).to.be('"SoCraTes Notifications" <' + supermanEmail + '>');
     });
 
     it('waitinglist - sends a meaningful mail to admins', () => {
-      notifications.changedWaitinglist(hans, {desiredRooms: [roomOptions.informationFor('junior', 'waitinglist')]});
+      const details = Object.assign({desiredRooms: [roomOptions.waitinglistInformationFor('junior'),
+        roomOptions.waitinglistInformationFor('bed_in_double')]}, roomOptions.nightsUntilFor(3));
```

Comment:
alles in eine Zeile? ich mag das nicht, aber wenn es euch lieber ist, dann eben in eine Zeile